frontend/src/actions/github/fetch_repos.js

```diff
@@ -1,16 +1,15 @@
-import { getThirdParty } from '../../lib/api';
+import { get } from '../../lib/api';
 import * as constants from '../../constants/github';
 
 
 /**
  * Fetch a user's repo
  */
 
-export default (username) => {
+export default (username, token) => {
   return dispatch => {
     dispatch(request(username));
-
-    return getThirdParty(`https://api.github.com/users/${username}/repos?per_page=100`, {params: {type: 'owner,member'}})
+    return get(`/../github/repositories/${token}`)
```

Comment:
@asood123 ok so the `api.get` method prepends whatever is in `API_ROOT` env variable. 
API_ROOT gives me localhost:3000/api not the api server but the website server and with `api/` as the parent resource.

I will create another `get` method so I can make requests for the api (localhost:/3060)